docs/user_guide/usage_patterns/data_transfer/transfer_relative_deadline/submit_transfer_relative_deadline.py

```diff
@@ -0,0 +1,42 @@
+import datetime
+
+import globus_sdk
+from globus_sdk.globus_app import UserApp
+
+# Tutorial Client ID - <replace this with your own client>
+NATIVE_CLIENT_ID = "61338d24-54d5-408f-a10d-66c06b59f6d2"
+USER_APP = UserApp("relative-deadline-transfer", client_id=NATIVE_CLIENT_ID)
+
+# Globus Tutorial Collection 1
+# https://app.globus.org/file-manager/collections/6c54cade-bde5-45c1-bdea-f4bd71dba2cc
+SRC_COLLECTION = "6c54cade-bde5-45c1-bdea-f4bd71dba2cc"
+SRC_PATH = "/home/share/godata/file1.txt"
+
+# Globus Tutorial Collection 2
+# https://app.globus.org/file-manager/collections/31ce9ba0-176d-45a5-add3-f37d233ba47d
+DST_COLLECTION = "31ce9ba0-176d-45a5-add3-f37d233ba47d"
+DST_PATH = "/~/example-transfer-script-destination.txt"
+
+
+def make_relative_deadline(offset: datetime.timedelta) -> str:
+    now = datetime.datetime.now(tz=datetime.UTC)
```

Comment:
As noted in another comment:

```suggestion
    now = datetime.datetime.now(tz=datetime.timezone.utc)
```